pkgs/top-level/python-packages.nix

```diff
@@ -22212,6 +22212,25 @@ in modules // {
     };
   };
 
+  yolk = buildPythonPackage rec {
```

Comment:
I really should always consult the current nixpkgs documentation, too much has changed from 15.09 …